### PR TITLE
OSDOCS-5158 adding admonition about storage known issue

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5158
https://issues.redhat.com/browse/OCPBUGS-7582

Link to docs preview:
https://59818--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-customizations.html#installation-gcp-enabling-confidential-vms_installing-gcp-customizations

QE review:
- [x] QE has approved this change.

Will open a corresponding PR for the 4.13 release notes: https://github.com/openshift/openshift-docs/pull/59822

### CODE REVIEW

**Comment by kalexand-rh** [comment]
LGTM

**Review by jianli-wei — APPROVED** [review]
/lgtm

**Comment by jianli-wei** [comment]
@bscott-rh I just noticed there's "[Technology Preview features](https://docs.openshift.com/container-platform/4.13/release_notes/ocp-4-13-release-notes.html#ocp-4-13-technology-preview)" section, so I suggest to add this 4.13 new feature into the table 13 _Installation Technology Preview features_, thanks!

File: modules/installation-gcp-enabling-confidential-vms.adoc [hunk 1]
```diff
@@ -18,6 +18,11 @@ You can use Confidential VMs when installing your cluster. Confidential VMs encr
 :FeatureName: Confidential Computing
 include::snippets/technology-preview.adoc[]
 
+[IMPORTANT]
+====
+Due to a known issue, you cannot create persistent volume storage on a cluster that uses Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
```
Comment (duanwei33) [inline modules/installation-gcp-enabling-confidential-vms.adoc:23]: PV will be created successfully but mount will fail.  So suggest update as below, wdyt?
 
```suggestion
Due to a known issue, you cannot use persistent volume storage on a cluster with Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
```

Comment (bscott-rh) [reply modules/installation-gcp-enabling-confidential-vms.adoc:23]: Thank you for this correction.

**Comment by bscott-rh** [comment]
> @bscott-rh I just noticed there's "[Technology Preview features](https://docs.openshift.com/container-platform/4.13/release_notes/ocp-4-13-release-notes.html#ocp-4-13-technology-preview)" section, so I suggest to add this 4.13 new feature into the table 13 _Installation Technology Preview features_, thanks!

Thank you, I will make this update in https://github.com/openshift/openshift-docs/pull/59822 where I update the other release notes sections.

**Comment by duanwei33** [comment]
LGTM

**Review by lpettyjo — COMMENTED** [review]
LGTM

**Pull request merged**